scripts/bump.sh

```diff
@@ -8,6 +8,9 @@ DEPENDENCIES=(
 
 # Variables
 NB_ARGS="$#"
+USAGE="bump.sh [-h | --help] [-p | --patch] [-m | --minor] [-M | --major]"
+HTTPS_REMOTE='https://github.com/iScsc/iscsc.fr.git'
```

Comment:
I found out why.
In my git remote, urls are with a lowercase `s`
I don't know if it's only for me, but we should do something about this
```suggestion
HTTPS_REMOTE='https://github.com/iscsc/iscsc.fr.git'
```
Another way could be to lowercase the url before the check

Reply:
or just use `grep --ignore-case` maybe? however that's weird... we should have the same git and remotes?

Reply:
Yes perfect!

Reply:
fixed :wink: